### PR TITLE
Prepare remove tx fee handler

### PR DESCRIPTION
**Pull request by miiu96**
- Integrated new `elrond-go-core`
- Removed `TransactionsFeeHandler`

### CODE REVIEW

**Review by sstanculeanu — DISMISSED** [review]

**Review by sstanculeanu — APPROVED** [review]

File: mock/dbTransactionsHandlerStub.go [hunk 1]
```diff
@@ -4,22 +4,22 @@ import (
...
 // DBTransactionProcessorStub -
 type DBTransactionProcessorStub struct {
-	PrepareTransactionsForDatabaseCalled func(body *block.Body, header coreData.HeaderHandler, pool *indexer.Pool) *data.PreparedResults
+	PrepareTransactionsForDatabaseCalled func(body *block.Body, header coreData.HeaderHandler, pool *outport.Pool) *data.PreparedResults
 	SerializeReceiptsCalled              func(recs []*data.Receipt, buffSlice *data.BufferSlice, index string) error
 	SerializeScResultsCalled             func(scrs []*data.ScResult, buffSlice *data.BufferSlice, index string) error
 }
 
-func (tps *DBTransactionProcessorStub) SerializeTransactionWithRefund(_ map[string]*data.Transaction, _ map[string]*data.RefundData, _ *data.BufferSlice, _ string) error {
+func (tps *DBTransactionProcessorStub) SerializeTransactionsFeeData(_ map[string]*data.FeeData, _ *data.BufferSlice, _ string) error {
```
Comment (bogdan-rosianu) [inline mock/dbTransactionsHandlerStub.go:17]: missing comment

File: process/transactions/serialize.go [hunk 2]
```diff
@@ -3,17 +3,14 @@ package transactions
...
 
-	elasticIndexer "github.com/ElrondNetwork/elastic-indexer-go"
 	"github.com/ElrondNetwork/elastic-indexer-go/converters"
 	"github.com/ElrondNetwork/elastic-indexer-go/data"
 	"github.com/ElrondNetwork/elrond-go-core/core"
-	"github.com/ElrondNetwork/elrond-go-core/data/transaction"
 )
 
-// SerializeScResults will serialize the provided smart contract results in a way that Elastic Search expects a bulk request
+// SerializeScResults will serialize the provided smart contract results in a way that ElasticSearch expects a bulk request
```
Comment (bogdan-rosianu) [inline process/transactions/serialize.go:13]: Elasticsearch

File: process/transactions/serialize.go [hunk 3]
```diff
@@ -31,7 +28,7 @@ func (tdp *txsDatabaseProcessor) SerializeScResults(scResults []*data.ScResult,
 	return nil
 }
 
-// SerializeReceipts will serialize the receipts in a way that Elastic Search expects a bulk request
+// SerializeReceipts will serialize the receipts in a way that ElasticSearch expects a bulk request
```
Comment (bogdan-rosianu) [inline process/transactions/serialize.go:31]: Elasticsearch

File: factory/indexerFactory.go [hunk 4]
```diff
@@ -132,9 +130,6 @@ func checkDataIndexerParams(arguments *ArgsIndexerFactory) error {
 	if check.IfNil(arguments.Hasher) {
 		return indexer.ErrNilHasher
 	}
-	if check.IfNil(arguments.TransactionFeeCalculator) {
-		return indexer.ErrNilTransactionFeeCalculator
```
Comment (bogdan-rosianu) [inline factory/indexerFactory.go:136]: remove this err from errors.go

File: process/elasticProcessor.go [hunk 5]
```diff
@@ -14,7 +14,7 @@ import (
 	"github.com/ElrondNetwork/elrond-go-core/core/check"
 	coreData "github.com/ElrondNetwork/elrond-go-core/data"
 	"github.com/ElrondNetwork/elrond-go-core/data/block"
-	"github.com/ElrondNetwork/elrond-go-core/data/indexer"
+	"github.com/ElrondNetwork/elrond-go-core/data/outport"
```
Comment (bogdan-rosianu) [inline process/elasticProcessor.go:17]: imports not sorted

**Review by bogdan-rosianu — CHANGES_REQUESTED** [review]
please run Inspect Code from your IDE. You will find some grammar issues, alongside with unsorted imports

File: mock/dbTransactionsHandlerStub.go (hunk 1 above)
Comment (miiu96) [reply mock/dbTransactionsHandlerStub.go:17]: added

File: process/transactions/serialize.go (hunk 2 above)
Comment (miiu96) [reply process/transactions/serialize.go:13]: done

File: process/transactions/serialize.go (hunk 3 above)
Comment (miiu96) [reply process/transactions/serialize.go:31]: done

File: factory/indexerFactory.go (hunk 4 above)
Comment (miiu96) [reply factory/indexerFactory.go:136]: done.

File: process/elasticProcessor.go (hunk 5 above)
Comment (miiu96) [reply process/elasticProcessor.go:17]: done.

**Review by bogdan-rosianu — APPROVED** [review]

**Review by sstanculeanu — APPROVED** [review]

**Pull request merged**